tests/sample.yaml

```diff
@@ -44,6 +44,11 @@ mgmt_port: 10090
 scylla_repo_m: 'http://repositories.scylladb.com/scylla/repo/f4a2920f80c4bf178217c2553ad65ad7/centos/scylladb-2017.1.repo'
 scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/branch-1.0/latest/scylla-manager.repo'
 
+# assign scylla repo, which will be used to install scylla or other related packages
+# scylla_repo: ''
+# assign scylla_repo for loader, in case we use CentOS as Loader when test Ubuntu/Debian as db nodes
+# scylla_repo_loader: ''
```

Comment:
use of old YAMLs configuration is discontinued, add the new param to `sct_config.py`